supertokens_python/recipe/passwordless/recipe.py

```diff
@@ -202,7 +212,7 @@ async def handle_api_request(
 
     async def handle_error(
         self, request: BaseRequest, err: SuperTokensError, response: BaseResponse
-    ):
+    ) -> BaseResponse:  # type: ignore
         raise err
```

Comment:
> Isn't it better to just return response instead of raise err

won't this suppress all errors?

> #type: ignore?
return type is BaseResponse but we aren't returning anything (raising an error instead)

Reply:
> #type: ignore

it expects BaseResponse but we don't return anything (raise err)

> just return response instead of raise err
won't that supress error?